codes/block_definitions/utilities/mutate_methods.py

```diff
@@ -70,7 +70,7 @@ def mutate_single_input(mutant_material: BlockMaterial, block_def): #: BlockDefi
                 break
 
 
-def mutate_single_ftn(mutant_material: BlockMaterial, block_def): #: BlockDefinition):
+def mutate_single_ftn(mutant_material: BlockMaterial, block_def, dont_exclude=False): #: BlockDefinition):
```

Comment:
Just added the ability to not exclude the current operator. Useful if there is only one possible operator, defaults to false to preserve backwards compatability